run_container.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/coreos/go-systemd/dbus"
+)
+
+// Public Functions
+
+// RunContainer runs a container with given ID
+func RunContainer(containerID string, containerPath string) error {
```

Comment:
@mrunalp That's not dissimilar to what I was originally thinking - but I'm not viewing this as part of the pod interface. There's an interface for talking directly to runc in ocid to get an individual container within a pod spooled up - that's what I was trying to produce here. The pod abstraction can be built once we have the ability to get containers started/stopped (unless you're saying we should skip containers and just try and handle whole pods at a time?)